runtime/executor/src/main/java/edu/snu/nemo/runtime/executor/MetricMessageSender.java

```diff
@@ -25,5 +25,7 @@
 
   void send(final String metricKey, final String metricValue);
 
+  void flush();
```

Comment:
Javadoc?

Reply:
I've added javadoc to all method. Thank you!